README.md

```diff
@@ -29,18 +31,24 @@ Create a `.env` file in the root of your project with the following variables:
 
 ```env
 # Your Rye API key (required)
-RYE_API_KEY=your_rye_api_key_here
-
+RYE_API_KEY=your_rye_staging_api_key_here
 # The base URL for the Rye API (required)
-RYE_BASE_URL=https://api.rye.com
-# use https://staging.api.rye.com to use for staging environment.
-
+RYE_BASE_URL=https://staging.api.rye.com
 # Stripe API Key (required)
-NEXT_PUBLIC_STRIPE_API_KEY=pk_live_51LgDhrHGDlstla3fOYU3AUV6QpuOgVEUa1E1VxFnejJ7mWB4vwU7gzSulOsWQ3Q90VVSk1WWBzYBo0RBKY3qxIjV00LHualegh
-# use `pk_test_51LgDhrHGDlstla3fdqlULAne0rAf4Ho6aBV2cobkYQ4m863Sy0W8DNu2HOnUeYTQzQnE4DZGyzvCB8Yzl1r38isl00H9sVKEMu` instead for staging environment
+NEXT_PUBLIC_STRIPE_API_KEY=pk_test_51LgDhrHGDlstla3fdqlULAne0rAf4Ho6aBV2cobkYQ4m863Sy0W8DNu2HOnUeYTQzQnE4DZGyzvCB8Yzl1r38isl00H9sVKEMu
+#
```

Comment:
empty lines mess up nextjs env var loading... took me a while to figure it out :-)